src/controllers/helpcontroller.js

```diff
@@ -0,0 +1,209 @@
+    {
+        var trip = new Trip([
+            {
+                sel: $(".factory"),
+                content: Controller.l("This is the factory, the place where all of your products are made."),
+                expose: true,
+                position: "e",
+                delay: 3000,
+            },
+            {
+                sel: $(".factory .tab-content"),
+                content: Controller.l("You can order specific amounts of each product. Try to add a product."),
+                expose: true,
+                position: "e",
+            },
+            {
+                sel: $(".factory .alert"),
+                content: Controller.l("Keep an eye on this info-box. It holds relevant information for the order you are placing. You need to have enough money (currently you have") + " " + GAME.model.config.money + ") " + Controller.l("and the order has to fit on the truck (maximum capacity is") + " " + GAME.model.config.orderCapacity + ").",
+                expose: true,
+                position: "e",
+                delay: 8000
+            },
+            {
+                sel: $(".factory button[type=submit]"),
+                content: Controller.l("If everything is satisfied you can place the order by clicking on the green button."),
+                expose: true,
+                position: "e",
+            }
+        ], {
+            delay: -1,
+            position: "e",
+            onEnd: function () {
+                if (intro) {
+                    HelpController.startFactoryOrdersTour(true);
+                }
+            }
+        });
+
+        trip.start();
+
+        // Next step when the input of the form has changed
+        $("form[name=newFactoryOrder] :input").change(function (e) {
+            trip.next();
+        });
+
+        // TODO Button becomes white, since the 'trip-exposed' class is overwriting Bootstrap's styles.
+        // This fixes it, but it's not really pretty. Should be a better way.
+        $(".factory button[type=submit]").css("background-color", "#5cb85c");
+
+        // Next step when clicked on the 'place order' button
+        $(".factory button[type=submit]").click(function (e) {
+            trip.next();
+        });
+    }
+
+    static startFactoryOrdersTour(intro = false)
+    {
+        var trip = new Trip([
+            {
+                sel: $("#factory-orders"),
+                content: Controller.l("This is where you can see all orders currently in transport to your warehouse."),
+                expose: true,
+                position: "e",
+            },
+            {
+                sel: $("#factory-orders .progress").first(),
+                content: Controller.l("The order will arrive in the warehouse, when the progress bar is fully green. Do make sure you clicked the 'play' button in the statusbar."),
+                expose: true,
+                position: "e",
+            }
+        ], {
+            delay: 5000,
+            onEnd: function () {
```

Comment:
Since we use babel now, arrow function can be used. No need to do so here, but then these things would resolve to 

```javascript
onEnd: () => {if (intro) HelpController.startWarehouseTour(true)}
```

I find that to be just beautiful.

Also, did I miss something or is `intro` not defined within scope?